### ISSUE TITLE
[Bug]: No way to ingest youtube or create transcripts in latest version installed via go

### ISSUE THREAD

**Issue by grimmolf** [danielmiessler/fabric#947]
### What happened?

ENVIRONMENT:  Red Hat Enterprise Linux release 9.4 (Plow)
GO VERSION: go version go1.23.1 linux/amd64

go install github.com/danielmiessler/fabric@latest installed fabric into $HOME/go/bin , but the version installed does not have the -y flag option, and neither yt nor ts apps were installed.

### Version check

- [X] Yes I was.

### Relevant log output

```shell
[grimm@localhost ~]$ fabric -h
Usage:
  fabric [OPTIONS]

Application Options:
  -p, --pattern=                    Choose a pattern
  -C, --context=                    Choose a context
      --session=                    Choose a session
  -S, --setup                       Run setup
      --setup-skip-update-patterns  Skip update patterns at setup
  -t, --temperature=                Set temperature (default: 0.7)
  -T, --topp=                       Set top P (default: 0.9)
  -s, --stream                      Stream
  -P, --presencepenalty=            Set presence penalty (default: 0.0)
  -F, --frequencypenalty=           Set frequency penalty (default: 0.0)
  -l, --listpatterns                List all patterns
  -L, --listmodels                  List all available models
  -x, --listcontexts                List all contexts
  -X, --listsessions                List all sessions
  -U, --updatepatterns              Update patterns
  -c, --copy                        Copy to clipboard
  -m, --model=                      Choose model
  -o, --output=                     Output to file
  -n, --latest=                     Number of latest patterns to list (default: 0)
  -d, --changeDefaultModel          Change default pattern

Help Options:
  -h, --help                        Show this help message

[grimm@localhost ~]$ fabric -y
unknown flag `y'
```

**Comment by derek3131** [comment]
Go towards the bottom of the README.md under the "Helper Apps" section.

yt Installation

To install yt, install it the same way as you install Fabric, just with a different repo name.

go install github.com/danielmiessler/yt@latest

Be sure to add your YOUTUBE_API_KEY to ~/.config/fabric/.env.

**Comment by grimmolf** [comment]
> Go towards the bottom of the README.md under the "Helper Apps" section.
> 
> 
> 
> yt Installation
> 
> 
> 
> To install yt, install it the same way as you install Fabric, just with a different repo name.
> 
> 
> 
> go install github.com/danielmiessler/yt@latest
> 
> 
> 
> Be sure to add your YOUTUBE_API_KEY to ~/.config/fabric/.env.

Ah, I see that. The README shows -y as an option, is that just not included yet, and is there still a way to get the ts helper app?

**Comment by coachdamon** [comment]
Having a similar problem here....I migrated from the python version to use go. Installed go with the package manager on silicon-based MBP. Added these into .zshrc (export GOROOT=/opt/homebrew/bin/go
export GOPATH=$HOME/go
export PATH=$GOPATH/bin:$GOROOT/bin:$HOME/.local/bin:$PATH:) but the go directory doesn't exist under /opt/homebrew/bin. 

This prevents me from installing yt, of course, but also makes me wonder if there's something else not configured correctly. Fabric seems to respond ok, but just checking. Any thoughts?
